config/auth.go

```diff
@@ -11,12 +11,12 @@ import (
 	"time"
 
 	"github.com/pkg/errors"
```

Comment:
suggestion: Is it possible to replace all existing `errors` to `scalingoerrors`? 
It will freed us from the use of another deprecated package

Reply:
It's not possible. We declare some public variable, for instance on line 24:

```go
var (
	ErrAuthenticationFailed = errors.New("authentication failed")
)
```

`scalingoerrors` requires a context and we have no context available here.